pdns/syncres.hh

```diff
@@ -498,6 +512,16 @@ public:
   {
     t_sstorage.ednsstatus.clear();
   }
+  static void pruneEDNSStatuses(time_t cutoff)
+  {
+    for (auto it = t_sstorage.ednsstatus.begin(); it != t_sstorage.ednsstatus.end(); ) {
+      if (it->second.modeSetAt <= cutoff) {
+        it = t_sstorage.ednsstatus.erase(it);
```

Comment:
Same question about the number of entries?

Reply:
On one of our small recursors (sub 1kqps) that has only been up for a week we have 200,000 edns status entries - if that data point helps.

Reply:
With cleanup, I suppose it will follow the same trend as the nsspeed size, which in that instance is an order of magnitude smaller.